src/components/ContentLayout/ContentContainer/ContentContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { View, ScrollView } from 'react-native';
+import { Props } from './types';
+import styles from './styles';
+
+const ContentContainer: React.FC<Props> = (props) => {
+  const { children, style, scroll = true, ...otherProps } = props;
+
+  const ContainerComponent = scroll ? ScrollView : View;
```

Comment:
ScrollView should not affect the footer, the reason for this refactor is to have the footer buttons always sticked to the bottom and visible. Everything other can be scrollable.

Reply:
I think I've missed that so in the new commit I added an attribute `fixed` for Header or Footer.
After setting _fixed_ to **true** components will stay in place if content is scrollable
in case of a Header it will always remain on top and in case of Footer it will always remain on the bottom